### DISCUSSION TITLE
What is the command to show or hide side bar panel

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-theia/theia#14066] [Q&A] [answered]
Hi,
   I want to show or hide side bar panel by code, Is there any command can do this?
![image](https://github.com/user-attachments/assets/576d5bf6-5e53-4ee1-9c0c-daabc0bdda5e)
![image](https://github.com/user-attachments/assets/528ee5ba-8139-49b0-adf0-5e14cc10ea70)

**Comment by msujew** [comment, marked as answer]
Hey @wss29,

each view has an associated "toggle" command. See also:

https://github.com/eclipse-theia/theia/blob/9c379c668fceb387ab36962408447fb5473d469c/packages/navigator/src/browser/navigator-contribution.ts#L175

Running this command toggles the view (i.e. hides it if it's visible or makes it visible if it's already hidden).

**Comment by wss29** [comment]
got it, thanks
